mozilla_version/fenix.py

```diff
@@ -7,6 +7,7 @@
 from mozilla_version.version import BaseVersion, VersionType
 from mozilla_version.parser import strictly_positive_int_or_none, positive_int
 
+# TODO remove in a future release - deprecated in favor of MobileVersion
```

Comment:
Is there a particular reason for copying instead of sharing code?

Reply:
I'm copying it so that I can phase FenixVersion out gradually, as per the previous comments, and then delete FenixVersion entirely. I don't see a reason to create a duplicate class but name it FocusVersion. Am I missing something?

Reply:
If the intention was to keep FenixVersion around, then yes, of course I would share code. But the intention is to delete it entirely once its use is changed everywhere.

Reply:
@jcristau Can we come to a resolution on this please. Its blocking me finishing shipit work :)

Reply:
Nm, aki explained :) pushed the change